### PR TITLE
Adjusted doctor/researcher tweezer/lemo spawn locations

### LINKED ISSUE #13180: If you don't take a backpack/satchel in your round start gear, you won't be given lemolime/tweezers as a MD or Researcher
### Description

Expected this stuff to just be dumped on the floor

### Test Merges

Round ID: [21542](https://scrubby.melonmesa.com/round/21542)
BYOND Version: 515.1608
Server revision compiled on: 
Master commit: [c194d42f06dde4bd91b15f23eee2e6b0b602060c](https://github.com/tgstation/TerraGov-Marine-Corps//commit/c194d42f06dde4bd91b15f23eee2e6b0b602060c)
The following pull requests are currently test merged:
[#12892: 'Adds a delay to throwing' by ivanmixo at commit a99898c494](https://github.com/tgstation/TerraGov-Marine-Corps//pull/12892)
[#12995: 'Laser rework' by Lumipharon at commit b4a58b8906](https://github.com/tgstation/TerraGov-Marine-Corps//pull/12995)
[#13095: 'Quickbuild Rework 2023' by RipGrayson at commit 4ce7da1f1f](https://github.com/tgstation/TerraGov-Marine-Corps//pull/13095)
[#13106: 'Silo scaling' by Lumipharon at commit 88aa066a4d](https://github.com/tgstation/TerraGov-Marine-Corps//pull/13106)
[#13136: 'Minimap improvement' by Lumipharon at commit c1c72870af](https://github.com/tgstation/TerraGov-Marine-Corps//pull/13136)
[#13155: 'pride month special' by tyeagg at commit 403ec53387](https://github.com/tgstation/TerraGov-Marine-Corps//pull/13155)

Local commit: 3c4b64881c2de3831812c7811cd401f7a1c8240e
Server tools version: 5.12.1

### Reproduction Steps

1. Go into gear customization
2. set 'no backpack'
3. spawn in as MD/Rsr
4. weep, as 50/52 recipes require lemolime

### Screenshots

_No response_

### PR DESCRIPTION
**Pull request by ivanmixo**
## About The Pull Request

The lemo bottle spawns in their suit storage and the tweezers spawn in their surgery pouch. Added surgery and medkit pouch to both MD and researcher as default spawns. 

Adds a new proc that tries to equip and failing that tries to put the item in the hands of the person spawning.

Closes #13180
## Why It's Good For The Game

Fix people who pick the no backpack pref being locked out of these for no reason.
## Changelog
:cl:
fix: Fixed researcher/md/cmo spawning without their lemo bottle and tweezers if they have the no backpack pref enabled
/:cl:

### CODE REVIEW

**Comment by ChocolateBickie** [comment]
Combat robots don't start with pouches or exosuits so this will just delete them for combat robot researchers.

**Comment by ivanmixo** [comment]
hampter

File: code/datums/jobs/job/shipside.dm [hunk 1]
```diff
@@ -804,17 +804,17 @@ Make sure that the doctors and nurses are doing their jobs and keeping the marin
 	glasses = /obj/item/clothing/glasses/hud/health
 	mask = /obj/item/clothing/mask/surgical
 	head = /obj/item/clothing/head/cmo
-	r_store = /obj/item/storage/pouch/medkit/medic
-	l_store = /obj/item/storage/pouch/surgery
+	r_store = /obj/item/storage/pouch/surgery
+	l_store = /obj/item/storage/pouch/medkit/medic
```
Comment (QualityVan) [inline code/datums/jobs/job/shipside.dm:808]: Why swap the sides?

Comment (ivanmixo) [reply code/datums/jobs/job/shipside.dm:808]: Consistency. Also I need to do work on this, I'll get around to it within the next couple days.

**Review by Lumipharon — APPROVED** [review]

**Pull request merged**